packages/liveblocks-core/src/room.ts

```diff
@@ -457,65 +459,190 @@ type SubscribeFn<
    * });
    */
   (type: "storage-status", listener: Callback<StorageStatus>): () => void;
+
+  (type: "comments", listener: Callback<CommentsEventServerMsg>): () => void;
 };
 
 export type GetThreadsOptions<M extends BaseMetadata> = {
   query?: {
     resolved?: boolean;
     metadata?: Partial<QueryMetadata<M>>;
   };
-  since?: Date;
 };
 
 type CommentsApi<M extends BaseMetadata> = {
+  /**
+   * Returns the threads within the current room and their associated inbox notifications.
+   * It also returns the request date that can be used for subsequent polling.
+   *
+   * @example
+   * const {
+   *   threads,
+   *   inboxNotifications,
+   *   requestedAt
+   * } = await room.getThreads({ query: { resolved: false }});
+   */
   getThreads(options?: GetThreadsOptions<M>): Promise<{
     threads: ThreadData<M>[];
     inboxNotifications: InboxNotificationData[];
-    deletedThreads: ThreadDeleteInfo[];
-    deletedInboxNotifications: InboxNotificationDeleteInfo[];
-    meta: {
-      requestedAt: Date;
+    requestedAt: Date;
+  }>;
+
+  /**
+   * Returns the modified and deleted threads and their associated inbox notifications since the requested date.
+   *
+   * @example
+   * const result = await room.getThreads();
+   * // ... //
+   * await room.getThreadsSince({ since: result.requestedAt });
+   */
+  getThreadsSince(options: { since: Date }): Promise<{
+    threads: {
+      modified: ThreadData<M>[];
```

Comment:
should we use `updated` instead of `modified`? To match `updatedAt`, we don't have `modifedAt`

Reply:
this would also hold for `getInboxNotificationsSince`